File: 1848-SumOfUniqueElements/1848-SumOfUniqueElements.js
// Last updated: 3/6/2026, 2:08:47 PM
/**
 * @param {number[]} nums
 * @return {number}
 */
var sumOfUnique = function(nums) {
    let obj = {};
    let sum = 0;
    for(let char of nums){
        if(obj[char]===undefined){
            obj[char]= 1;
        }
        else{
            obj[char]++;
        }
    }
   for(let value in obj){
       if(obj[value] === 1){
           sum = sum + Number(value);
       }
   }
   return sum;
};

console.log(sumOfUnique([1,2,3,2]))

File: 1848-SumOfUniqueElements/1848-SumOfUniqueElements.cpp
// Last updated: 7/17/2026, 4:26:00 PM
class Solution {
public:
    int sumOfUnique(vector<int>& nums) {
        vector<int> freq(101, 0);

        for (int num : nums) {
            freq[num]++;
        }

        int sum = 0;

        for (int num : nums) {
            if (freq[num] == 1) {
                sum += num;
            }
        }

        return sum;
    }
};